trunk/NDHMS/Land_models/NoahMP/phys/module_sf_noahmplsm.F

```diff
@@ -400,7 +400,7 @@ SUBROUTINE NOAHMP_SFLX (parameters, &
 		   SHG     , SHC     , SHB     , EVG     , EVB     , GHV     , & ! OUT :
 		   GHB     , IRG     , IRC     , IRB     , TR      , EVC     , & ! OUT :
 		   CHLEAF  , CHUC    , CHV2    , CHB2    , FPICE   , PAHV    , &
-                   PAHG    , PAHB    , PAH                                     &
+                   PAHG    , PAHB    , PAH     , LAISUN  , LAISHA  , RB        & ! OUT
```

Comment:
OUT! 
are all those variables out? they were not previously labeled as such.
This is perty minor...